MacrobenchmarkSample/app/src/main/java/androidx/benchmark/macro/target/RecyclerViewActivity.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.benchmark.macro.target
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+
+class RecyclerViewActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        title = "RecyclerView Sample"
+        setContentView(R.layout.activity_recycler_view)
+        val recycler = findViewById<RecyclerView>(R.id.recycler)
+        val itemCount = intent.getIntExtra(EXTRA_ITEM_COUNT, 1000)
```

Comment:
add a comment:

This argument allows the macrobenchmark to define the content being tested, and test different scenarios. In your app, you could use approach to navigate to consistent UI content, e.g. with consistent generated list items